provider/resource_edge_deployment_service_update_backend.go

```diff
@@ -0,0 +1,51 @@
+package provider
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+func resourceEdgeDeploymentServiceBackend() *schema.Resource {
+	return &schema.Resource{
+		Create:   updateEdgeDeploymentServiceBackend,
+		Read:     readEdgeDeploymentServiceBackend,
+		Update:   updateEdgeDeploymentServiceBackend,
+		Delete:   deleteEdgeDeploymentServiceBackend,
+		Importer: &schema.ResourceImporter{},
+		Schema: map[string]*schema.Schema{
+			"site_short_name": {
+				Type:        schema.TypeString,
+				Description: "Site short name",
+				Required:    true,
+			},
+
+			"fastly_sid": {
+				Type:        schema.TypeString,
+				Description: "Fastly service ID",
+				Required:    true,
+			},
+
+			"fastly_service_vcl_active_version": {
```

Comment:
@BrooksCunningham this doesn't appear to be used?

Reply:
@BrooksCunningham I removed the unused variable.

Reply:
@sjparkinson that seems possible -- I'll add it back in case that was the original intention